### PR TITLE
11.0 develop technical sheet

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/i18n/fr.po [hunk 1]
```diff
@@ -959,64 +1237,71 @@ msgid "Spectators"
 msgstr "Spectateurs"
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:230
+#: code:addons/project_event/models/project_task.py:249
 #, python-format
-msgid "Spectators value should be in the range["
-msgstr "Le nombre de spectateurs doit être compris entre ["
+msgid "Spectators value should be in the range[-]"
+msgstr "Spectators value should be in the range[-]"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: French translation missing

File: project_event/i18n/fr.po [hunk 2]
```diff
@@ -959,64 +1237,71 @@ msgid "Spectators"
...
 #: model:ir.model.fields,field_description:project_event.field_project_task_task_state
+#:  
 #: model:ir.model.fields,field_description:project_event.field_project_task_task_state_report_done_required
+#:  
 #: model:ir.model.fields,field_description:project_event.field_project_task_task_state_report_not_done_required
-#: model:ir.ui.view,arch_db:project_event.project_event_form
+#:  model:ir.ui.view,arch_db:project_event.project_event_form
 msgid "State"
-msgstr "État"
+msgstr "Région"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: State is État in this case

File: project_event/i18n/fr.po [hunk 3]
```diff
@@ -1093,17 +1379,17 @@ msgstr ""
 
 #. module: project_event
 #: model:ir.model.fields,field_description:project_event.field_project_task_responsible_id
-msgid "Responsible"
-msgstr "Responsable de tâche/activité"
+msgid "Task/Activity Responsible"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po:1382]: Check if we have to keep the old (responsible) or the new one (Task/Activity Responsible)

File: project_event/i18n/fr.po [hunk 4]
```diff
@@ -1281,7 +1582,13 @@ msgid "res.partner.sector"
...
-msgid "The task should be related to a project!"
-msgstr "La tâche doit être reliée à un projet"
+msgid "this resource is not bookable"
+msgstr "Cette ressource n'est pas réservable"
+
+#. module: project_event
+#: code:addons/project_event/models/project_task.py:382
+#, python-format
+msgid "this room is not bookable"
+msgstr "Cette salle n'est pas réservable"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: new line at the end

File: project_event/i18n/fr.po [hunk 5]
```diff
@@ -393,77 +540,99 @@ msgid "Document Followers"
...
+#: model:ir.ui.menu,name:project_event.project_event_sub_menu 
+#: model:ir.ui.view,arch_db:project_event.project_activity_search 
+#: model:ir.ui.view,arch_db:project_event.project_event_activity_calendar 
+#: model:ir.ui.view,arch_db:project_event.project_event_activity_form 
+#: model:ir.ui.view,arch_db:project_event.project_event_activity_tree 
+#: model:ir.ui.view,arch_db:project_event.project_event_form 
 #: selection:project.project,project_type:0
 msgid "Event"
-msgstr "Événement"
+msgstr "Évènement"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: Événement

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/i18n/fr.po (hunk 1 above)
Comment (eilst) [reply project_event/i18n/fr.po]: Done

File: project_event/i18n/fr.po (hunk 2 above)
Comment (eilst) [reply project_event/i18n/fr.po]: Done

File: project_event/i18n/fr.po (hunk 5 above)
Comment (eilst) [reply project_event/i18n/fr.po]: Done

File: project_event/i18n/fr.po (hunk 3 above)
Comment (eilst) [reply project_event/i18n/fr.po:1382]: Done

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**